nav2_map_server/src/map_server.cpp

```diff
@@ -76,15 +76,16 @@ void MapServer::getParameters()
       map_filename_ = std::string(dirname(fname_copy)) + '/' + map_filename_;
       free(fname_copy);
     }
-  } catch (YAML::Exception) {
-    std::string msg = "'" + yaml_filename_ + "' does not contain an image tag or it is invalid";
+  } catch (YAML::Exception & e) {
+    std::string msg = "'" + yaml_filename_ + \
```

Comment:
The backslash at the end of the line shouldn't be necessary.

Reply:
Shouldnt be necessary, but a typical IDE when that is present will comment out all lines that are continued with the `\` so it helps in debugging.

Reply:
I can remove if you like.

Reply:
I dislike line continuation characters but if it plays nicer with the tools, that's a good reason to keep them. 

However, the logical conclusion would be every long line that is split should have a line continuation character at the end. We have a lot of them. Is that really where we want to go with the code?

Reply:
Generally I do put them everywhere, but I did this one pretty fast, I'll just remove it to keep it all consistent